nature/forms.py

```diff
@@ -3,6 +3,21 @@
 
 
 class FeatureForm(forms.ModelForm):
+    name = forms.CharField(
+        widget=forms.Textarea,
+        max_length=80,
```

Comment:
Make name a text field might be misleading as it only allow 80 chars, would it be better to make the input field a bit longer?